src/app/routes/regex/index.js

```diff
@@ -11,13 +11,20 @@ const assetUriRegex = '[a-z-_]{0,}[0-9]{8,}';
 
 const variantRegex = '/simp|/trad|/cyr|/lat';
 
-export const articleRegexPath = `/:service(${serviceRegex})/articles/:id(${idRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
+const articleLocalMapping = {
+  cymrufyw: 'erthyglau',
+  naidheachdan: 'sgeulachdan',
+};
+
+const articleLocalRegex = `articles|${articleLocalMapping.cymrufyw}|${articleLocalMapping.naidheachdan}`;
+
+export const articleRegexPath = `/:service(${serviceRegex})/:local(${articleLocalRegex})/:id(${idRegex}):variant(${variantRegex})?:amp(${ampRegex})?`;
 
 export const articleDataRegexPath = `${articleRegexPath}.json`;
 
-export const articleSwRegexPath = `/:service(${serviceRegex})/articles/sw.js`;
+export const articleSwRegexPath = `/:service(${serviceRegex})/:local(${articleLocalRegex})/sw.js`;
```

Comment:
```suggestion
export const articleSwRegexPath = `/:service(${serviceRegex})/${articlePathRegex}/sw.js`;
```

Reply:
I tried that, but react-router doesn't seem to understand the regex without `:param(regex)`.

Reply:
I think this might work if you add parentheses to the `articlePathRegex` variable